src/components/NotificationItem/styles.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+import CloseIcon from '@material-ui/icons/Close';
+import SEVERITY from '../../types/Severity';
+
+export const Container = styled.div<{severity: SEVERITY}>`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border-radius: 10px;
+    background-color: ${({severity, theme}) => 
+        severity === SEVERITY.Error? theme.colors.error
+        : severity === SEVERITY.Warning? theme.colors.warning: theme.colors.secondaryLight};
+    color: ${({ severity, theme }) => severity === SEVERITY.Warning? theme.colors.blueActive : theme.colors.whiteFontColor};
+    box-shadow: ${({ theme }) => theme.boxShadow};
+    width: 20rem;
+    max-height: 4rem;
+    margin-bottom: 0.6rem;
+    svg {
+        font-size:1rem;
+    }
+`
+
+export const Close = styled(CloseIcon)`
+    margin-top: 5px;
```

Comment:
cursor: pointer to every item that you can click on!

Reply:
fixed